set-client-paths.sh

```
@@ -9,9 +9,9 @@ load-env
 
 client_prefs="$HOME/.faforever/client.prefs"
 
-jq --arg game_path "$game_path" \
+jq --arg game_path "$(readlink -f "$game_path")" \
```

Shellcheck (0.11.0)

[warning] game_path is referenced but not assigned.

(SC2154)